saltysimulations/salty-747
# fix(AP) Fixed TO/GA keybind.

Fixes an inconsistency with TO/GA logic depending on whether the keybind or the cockpit switch is used to trigger it.

Also fixes TO/GA wrongly engaging the AT in SPD mode.

### Testing instructions:

Spawn on runway, complete 1 takeoff using keybind, and 1 with the TOGA switch, with armed VNAV. 
Note correct AT FMA indications THR REF -> HOLD (at 65kts) -> THR REF (at 400 feet) -> SPD (at level off altitude).

Set up an ILS approach, when established and stable on glideslope perform go-around using keybind and check 
THR REF | TO/GA | TO/GA correctly displays, thrust is applied and the aircraft climbs suitably to the set MCP alt.
